jenkins-pipelines/rolling-upgrade-ami.jenkinsfile

```diff
@@ -5,7 +5,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingUpgradePipeline(
     backend: 'aws',
-    base_versions: ['4.4'],
+    base_versions: ['2020.1', '4.3', '2021.1'],
```

Comment:
why do you need to upgrade from '2021.1' to the same version, '2021.1'?

Reply:
The `2021.1` in base_version is official released (2021.1.x), the destination version will be unstable 2021.1.y

Reply:
got it